shuup/reporting/admin_module/forms.py

```diff
@@ -0,0 +1,68 @@
+from django import forms
+from django.forms import DateField
+from django.utils.encoding import force_text
+from django.utils.translation import ugettext_lazy as _
+
+from shuup.core.models import Shop
+from shuup.reporting.reporter.base import (
+    get_report_class, get_report_classes, get_writer_names
+)
+from shuup.reporting.utils import parse_date_range
+
+
+class ReportForm(forms.Form):
+
+    def __init__(self, **kwargs):
+        super(ReportForm, self).__init__(**kwargs)
```

Comment:
Just a suggestion but would like to see filter based on customer, orderer and creator.